sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -75,7 +75,9 @@ AZ_NODISCARD az_iot_provisioning_client_options az_iot_provisioning_client_optio
  * @param[in] global_device_hostname The device provisioning services global host name.
  * @param[in] id_scope The ID Scope.
  * @param[in] registration_id The Registration ID. This must match the client certificate name (CN
- * part of the certificate subject).
+ * part of the certificate subject). Must be alphanumeric characters plus the special characters:
```

Comment:
Can we point to the Docs page instead (or in addition)? 

https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#registration-id

I think I've been through at least one change in requirements and if this changes again, it's unlikely we'll know to change our comment.

Reply:
In fairness though, our implementation would have to change if that were the case....documentation won't change that.